### ISSUE TITLE
unknown service: myhome.sync_time

### ISSUE THREAD

**Issue by stepir** [OpenWebNet-HA/MyHOME#82]
Hi

I had this automation working since ever - but starting after the recent new versions I noticed this popup coming every day. Note that I run it every 4hours but only once it give this error:

```
MHf454 Timesync uses an unknown service
The automation "MHf454 Timesync" (automation.mhf454_timesync) has an action that calls an unknown service: myhome.sync_time.

This error prevents the automation from running correctly. Maybe this service is no longer available, or perhaps a typo caused it.

To fix this error, [edit the automation](http://192.168.2.242:8123/config/automation/edit/1636144760828) and remove the action that calls this service.

Click on SUBMIT below to confirm you have fixed this automation.
```

![image](https://user-images.githubusercontent.com/36472935/229596681-1d17fe41-c0be-45eb-b309-99ed2e53de93.png)

any idea why? If I manually run the automation it works:

```
alias: MHf454 Timesync
description: ""
trigger:
  - platform: time_pattern
    hours: /1
condition: []
action:
  - service: myhome.sync_time
    data: {}
mode: single

```

**Comment by sxpert** [comment]
I had something similar with something else when restarting the integration... services will disappear for some time, and if the automation starts right at the wrong moment, it won't be able to find the service.

**Comment by GreenGrassBlueOcean** [comment]
Thank you for your contribution.

We are closing this ticket as part of the transition to the **[OpenWebNet-HA](https://github.com/OpenWebNet-HA)** community repository. This issue relates to legacy YAML configuration (`myhome.yaml`) or older Home Assistant versions that have been superseded by modern UI-based configuration and dynamic auto-discovery.

If you are running the modern integration on Home Assistant 2024+ and still encounter an issue, please feel free to open a fresh ticket with updated logs.
